src/database/config/fakeData.sql

```diff
@@ -8,7 +8,8 @@ VALUES
 INSERT INTO users (full_name, username, email, password, avatar, is_active, career_status_id)
 VALUES
   ('John Doe', 'johndoe', 'johndoe@example.com', 'password123', 'avatar1.jpg', true, 1),
-  ('Jane Smith', 'janesmith', 'janesmith@example.com', 'password456', 'avatar2.jpg', true, 2);
+  ('Jane Smith', 'janesmith', 'janesmith@example.com', 'password456', 'avatar2.jpg', true, 2),
+  ('Admin', 'admin', 'codeacademy.gsg@gmail.com', '$2b$10$EXkKb.e1TQrSJkoGnbQ2nO5.V2uWuIOeLqwICylJEj5cG1BTXblBa', 'avatar2.jpg', true, 2);
```

Comment:
admin must be in bulid file so remove it from fakedata
note:i added it in my last branch